utils/src/ipfs.ts

```diff
@@ -0,0 +1,12 @@
+import { create } from 'ipfs-http-client';
+
+const ipfsEndpoint = 'https://ipfs-api.dev.fleek.cool';
+
+export const createIpfs = (fleekApiKey: string) => {
+  return create({
+    url: ipfsEndpoint,
+    headers: {
+      Authorization: `v2 ${fleekApiKey}`,
+    },
+  });
+};
```

Comment:
Hmm do wanna move ipfs.ts into here and use that ?
https://github.com/dcgtc/dgrants/blob/main/app/src/utils/ipfs.ts#L4-L15

Reply:
I'm confused @thelostone-mc, I did update that file as well: https://github.com/dcgtc/dgrants/pull/228/files#diff-281976158b6ff6d4d2122dab6c7b5ed9119e76f5a9181fc9c999698335b6692aR6

Reply:
Oh derp derp ! didn't see this! I messed up :P